sample-code/spring-app/src/test/java/com/sap/ai/sdk/app/controllers/OrchestrationTest.java

```diff
@@ -137,4 +144,29 @@ void testMaskingPseudonymization() {
         .doesNotContain("MASKED_PERSON")
         .contains("Mallory");
   }
+
+  @Test
+  @DisplayName("Declared OrchestrationAiModels must match Orchestration's list of available models")
+  @SneakyThrows
+  void orchestrationModelAvailability() {
+    // TODO: Change this test to be like ScenarioTest.openAiModelAvailability() once the
+    //  Orchestration service has made the "available models endpoint".
+    //  Right now this test cannot tell if we are lacking models.
```

Comment:
Previously, if we put a bad model name, orchestration would tell you the list of available models, now, it says:
```
400 Bad Request: 'Streaming is not supported for this model'
```
Orchestration needs to fix this

Reply:
I don't understand, does this mean the test is not working (yet)?

Reply:
Do we have a backlog item?

Reply:
I need to investigate why `// TODO` was not flagged in the pipeline :o

Reply:
I created AI/ai-sdk-java-backlog#114
Also the pipeline doesn't check todos in tests